### ISSUE TITLE
Q&A: How to prevent overlapping nodes being merged when dragging ?

### ISSUE THREAD

**Issue by holgerbrandl** [ArsMasiuk/qvge#177]
Hi there, when using the editor, I often struggle with overlapping shapes being unintentionally merged automatically. 

During dragging the left node
![image](https://github.com/ArsMasiuk/qvge/assets/200952/09561502-8d2e-4bdc-8dbd-cd385925d7f0)
 
After releasing the mouse: the entire node has disappeared:
![image](https://github.com/ArsMasiuk/qvge/assets/200952/1278ee9e-e444-439c-a9ac-a371bb96c20d)

Is there a ways (modifier key or similar) to prevent auto-merging of overlapping nodes?

**Comment by ArsMasiuk** [comment]
Hi, you mean this happens after you have released the mouse button while
dragging over a node? This is intended behaviour, otherwise nodes would
appear one inside another or stacked.

Holger Brandl ***@***.***> schrieb am Mo., 22. Jan. 2024,
08:00:

**Comment by holgerbrandl** [comment]
Yes, when releasing the mouse while dragging node (with some partial intersection with another node).

I do not think that partially stacking nodes is necessarily a bad/incorrect function. From a graph perspective it is legit. Clearly from a UI perspective it becomes harder to manage.

With the current auto-merge, it is impossible to place nodes with partial intersection (although this would be correct from a domain perspective (here a transport network). The orange rectangle will disappaer when releasing the mouse
![image](https://github.com/ArsMasiuk/qvge/assets/200952/b244a459-c9cc-4ca5-be67-05b7575862ed)

Workaround: Move nodes with the keyboard, but this is quite tedious.

Could potentially a modifier key support be added to prevent auto-merging?

**Comment by ArsMasiuk** [comment]
Well; I see... This feature can be useful indeed.
I'll put it on 0.7 release, hopefully it will be released sooner or later.
Thank you for using the Qvge btw :)

**Comment by holgerbrandl** [comment]
Looking forward to the release!

**Comment by ArsMasiuk** [comment]
Done for 0.7 dev now.

**Comment by holgerbrandl** [comment]
Great support. Thanks!
